execution_engine/src/storage/trie_store/operations/mod.rs

```diff
@@ -241,7 +246,11 @@ where
     E: From<S::Error> + From<bytesrepr::Error>,
 {
     let mut missing_descendants = Vec::new();
+    let mut visited = HashSet::new();
     while let Some(trie_key) = trie_keys_to_visit.pop() {
+        if !visited.insert(trie_key.clone()) {
```

Comment:
No need to clone, says Clippy.

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/1074/commits/99742d03286e7e825ef94537ef644bd016db9ec6